core/src/main/java/io/wcm/testing/mock/aem/MockTag.java

```diff
@@ -348,12 +348,37 @@ public Map<Locale, String> getLocalizedTitlePaths() {
 
   @Override
   public String getTitlePath() {
-    throw new UnsupportedOperationException("Unsupported operation");
+    return getTitlePath(null);
   }
 
   @Override
   public String getTitlePath(Locale arg0) {
```

Comment:
please move both getTitlePath methods above the "--- unsupported operations ---" comment once implemented

Reply:
Moved, thanks for noticing :)